components/dashboard/src/admin/BlockedRepositories.tsx

```diff
@@ -148,8 +148,10 @@ export function BlockedRepositoriesList(props: Props) {
                 </div>
             </div>
 
-            <Alert type={"info"} closable={false} showIcon={true} className="flex rounded p-2 w-2/3 mb-2 w-full">
-                <span>Search entries by their Repositoriy URL regular expression (RegEx).</span>
+            <Alert type={"info"} closable={false} showIcon={true} className="flex rounded p-2 mb-2 w-full">
```

Comment:
Here there was an extra class which was not used because `w-full` already changed the width.